src/libs/actions/Session/index.js

```diff
@@ -252,7 +252,7 @@ function beginSignIn(login) {
         },
     ];
 
-    API.read('BeginSignIn', {email: login}, {optimisticData, successData, failureData});
+    API.read('BeginSignIn1', {email: login}, {optimisticData, successData, failureData});
```

Comment:
Is this a typo? Or did we actually create a new command named `BeginSignIn1`? What problem are we trying to solve by doing that?

Reply:
so sorry I forgot to link the related Web-E PR for this.

the notification for this weird naming is [explained on the OP of that PR](https://github.com/Expensify/Web-Expensify/pull/37507#issue-1716210807).

Reply:
@marcaaron The Web-E PR and the App PR technically need to go out simultaneously–with just the Web-E PR the sign up flow would technically be broken for all clients. So here's the roll out plan:

1. Deploy a Web-E PR with `BeginSignIn1` that isn't called anywhere yet. https://github.com/Expensify/Web-Expensify/pull/37507
2. In this App PR, call `BeginSignIn1`
3. In another Web-E PR, update `BeginSignIn` = `BeginSignIn1`
4. In another App PR, update to call `BeginSignIn`
5. In final Web-E PR, delete `BeginSignIn1`.

Reply:
>Thoughts? I couldn't really think of any other way. Really didn't wanna be temporarily breaking the sign-up flow since it's so widely-used.

Hmm can someone summarize what the change is? And why we are doing the rollout this way?

It's hard for me to comment further because I don't actually understand what problem we're trying to solve by creating a new command. 

On first blush, changing the command name seems less preferable to adding a param that we can remove later once the old clients stop calling `BeginSignIn` without that param.

e.g. have new clients send `isNewBeginSignInFlow: true` then have Web-E run either the deprecated flow or the new flow depending on if the param is sent. Eventually we can just remove the deprecated code and stop sending the param after verifying there are no more "old clients". 

I am suggesting this way because we are just going to end up with a similar problem where some clients will still be sending `BeginSignIn1` when we want to delete this one later? Or maybe I am missing it. LMK!

Reply:
> we are just going to end up with a similar problem where some clients will still be sending BeginSignIn1 when we want to delete this one later

this is true although we're planning to remove `BeignSignIn1` after waiting for a while

if we add `isNewBeginSignInFlow: true`, we also have to (or want to) clean up this and related condition statements eventually and we have to determine when it's a good time to remove it

> have new clients send `isNewBeginSignInFlow: true` then have Web-E run either the deprecated flow or the new flow depending on if the param is sent.

this seems to be more clean way to do what we were planning to do

so the revised rollout plan will be
1. Modify Web-E PR. `isNewBeginSignInFlow` is `true`, call `UserAPI:newBeginSingIn` instead of `UserAPI::beginSignIn`. 
2. In this App RP, modify the code to send the additional parameter `isNewBeginSignInFlow` to the backend
3. Down the line, remove `isNewBeginSignInFlow` parameter check and `UserAPI::newBeginSingIn` by default. Stop sending `isNewBeginSignInFlow` to API from NewDot

Reply:
@jasperhuangg what do you think? If you agree, I gonna modify the existing PRs.

Reply:
I agree! Seems simpler than adding a whole new API command, and essentially achieves the same thing, do it!